src/org/jitsi/impl/neomedia/transform/AbsSendTimeEngine.java

```diff
@@ -129,8 +130,66 @@ private void setTimestamp(byte[] buf, int off)
      * disable this transformer.
      * @param id the ID to set.
      */
-    public void setExtensionID(int id)
+    public void setExtensionID(byte id)
     {
         extensionID = id;
     }
+
+    /**
+     *
+     * @param packet is the RawPacket
+     * @returns the Absolute Send Time header extension if present
+     */
+    public HeaderExtension hasAbsoluteSendTimeExtension(RawPacket packet)
+    {
+
+        if(packet.getExtensionBit() )
```

Comment:
We should also check if we have a valid ID here (RawPacket.getHeaderExtension() will return null if we ask for -1, but it will still loop through the extensions which isn't necessary).